### PR TITLE
Added Golang solutions + fixed format inconsistencies

### PR DESCRIPTION
**Pull request by blurout**
# =]

### CODE REVIEW

**Comment by mitchellirvin** [comment]
Hi @blurout ! Sorry it's taken so long to review this. Do you mind updating to handle the merge conflcits?

**Comment by blurout** [comment]
> 

No problem @mitchellirvin , I'll add a couple more solutions as well before I do so.

**Comment by blurout** [comment]
@mitchellirvin Done. Should be good to go.

File: go/Blind 75/arrayAndHashing/15-3Sum.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
+func threeSum(nums []int) [][]int {
```
Comment (mitchellirvin) [inline go/Blind 75/arrayAndHashing/15-3Sum.go:1]: can we de-dupe the `arrayAndHashing` and `Arrays & Hashing` dirs?

File: go/Blind 75/arrayAndHashing/217-Contains-Duplicate.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package arrayAndHashing
```
Comment (mitchellirvin) [inline go/Blind 75/arrayAndHashing/217-Contains-Duplicate.go:1]: nit: do you mind matching the naming convention for all these files & folders to be the same as the cpp dir? https://github.com/neetcode-gh/leetcode/tree/main/cpp/neetcode_150

the dir and file names should all be `snake_case`

**Review by mitchellirvin — CHANGES_REQUESTED** [review]
a few more small things, thanks @blurout !

**Pull request merged**